extensions/ghc_version.bzl

```diff
@@ -0,0 +1,15 @@
+""" Module extension which configures the GHC version"""
```

Comment:
Correct, this will only effect the default GHC version used, and only for rules_haskell and rules_haskell_tests. Other modules are not effected at all. They will always just get the default GHC version if using `version = None` and the specified version otherwise.